Velaptor/Reactables/MouseButtonReactable.cs

```diff
@@ -16,7 +16,7 @@ namespace Velaptor.Reactables
     internal class MouseButtonReactable : Reactable<(MouseButton button, bool isDown)>
     {
         /// <summary>
-        /// Sends a push notification to signal mouse button state changes.
+        /// Sends a push notification to signal a change to a mouse button.
```

Comment:
Refactor comment to below:

```suggestion
        /// Sends a push notification to signal a change to the state of a mouse button.
```